sormas-backend/src/main/java/de/symeda/sormas/backend/infrastructure/community/CommunityFacadeEjb.java

```diff
@@ -286,6 +287,15 @@ public CommunityDto save(@Valid CommunityDto dto, boolean allowMerge) throws Val
 		if (!featureConfiguration.isFeatureEnabled(FeatureType.EDIT_INFRASTRUCTURE_DATA)) {
 			throw new ValidationRuntimeException(I18nProperties.getValidationError(Validations.infrastructureDataLocked));
 		}
+		if (dto.getUuid() != null) {
```

Comment:
Please
* remove the unnecessary nesting (i.e. combine the two statements in the if and the else block)
* move the checks to below the getByUuid methods and check whether the entity is null or not; DTOs will always have a UUID when they come from the UI, so the current logic will not work properly